packages/extra/ingress/README.md

```diff
@@ -13,4 +13,5 @@
 | `dashboard`       | Should ingress serve Cozystack service dashboard                  | `false` |
 | `cdiUploadProxy`  | Should ingress serve CDI upload proxy                             | `false` |
 | `virtExportProxy` | Should ingress serve KubeVirt export proxy                        | `false` |
+| `api`             | Should ingress serve Cozystack API                                | `true`  |
```

Comment:
I think it's safer not to expose the k8s API by default. Let's leave it as an opt-in feature?
```suggestion
| `api`             | Should ingress serve Cozystack API                                | `false`  |
```